core-rust/mesh-api-server/src/mesh_api/handlers/network_options.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{mesh_api::generated::SCHEMA_VERSION, prelude::*};
+
+pub(crate) async fn handle_network_options(
+    state: State<MeshApiState>,
+    Json(request): Json<models::NetworkIdentifier>,
+) -> Result<Json<models::NetworkOptionsResponse>, ResponseError> {
+    assert_matching_network(&request.network, &state.network)?;
+
+    let database = state.state_manager.database.snapshot();
+
+    let mut bundles_iter = database.get_committed_transaction_bundle_iter(StateVersion::of(1));
```

Comment:
DId try it originally but was receving this error:
```
thread 'tokio-runtime-worker' panicked at state-manager/src/store/rocks_db.rs:780:17:
DB inconsistency! transaction version (1) doesn't match expected state version (0)
```

Reply:
Also in the `core-api` there are some constraints (must be >= 1) when extracting state version. See [this](https://github.com/radixdlt/babylon-node/blob/f67e4b50042a2a8cf0d1dff6f229a347d78edeeb/core-rust/core-api-server/src/core_api/conversions/numerics.rs#L195).

Reply:
Ahh fair enough. That's because `get_committed_transaction_bundle_iter` is designed for the transaction stream and very heavy.

It would be better to use `database.get_proof_iter(StateVersion::pre_genesis())` and check for `proof.ledger_header.proposer_timestamp_ms`.